tests/test_geocoder_standalone.py

```
@@ -0,0 +1,336 @@
+"""
+
+import unittest
+import time
+import sys
+import os
+
+# Add the standalone components to test
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'scripts'))
+
+# Import the core components from our standalone demo
+exec(open('scripts/geocoder_demo_standalone.py').read())
```

Ruff (0.12.2)

Use of `exec` detected

(S102)